widgetssdk/src/main/java/com/glia/widgets/core/callvisualizer/domain/CallVisualizer.java

```diff
@@ -6,16 +6,30 @@
 
 /**
  * Provides controls related to Call Visualizer module
- * TODO: improve docs in MOB-1816
+ *
+ * Some interactions between operators and visitors start as regular phone calls with no Glia engagement.
+ * Screensharing and video engagement are available for such calls via Call Visualizer.
+ *
+ * Please refer for our detailed guide on how to use Call Visualizer in our developers portal:
+ * TODO: add a link to Call Visualizer integration guide
  */
 public interface CallVisualizer {
     /**
-     * This builds a VisitorCodeView component that can be integrated in the client application
+     * Creates a VisitorCodeView component that can be integrated in the client application
```

Comment:
It seems to me that "into" should be used instead of "in" here:
can be integrated in the client application => can be integrated into the client application